### PR TITLE
DEBUG: Investigate failing tests in datalad-extensions

### PR DESCRIPTION
**Pull request by bpoldrack**
Addressing #158.

### CODE REVIEW

**Comment by bpoldrack** [comment]
So this is strange. Just dropping some snippets here to not forget about them while digging

```
[INFO] ...>aggregate:370,757>log:297  Finished metadata aggregation into Dataset(/tmp/datalad_temp__single_session_dicom2bids5fgc8gsb) 
[INFO] ...>test_dicomconv:56>dataset:498>utils:494,482,413,552>spec2bids:268>utils:413,552>run_procedure:442  Running procedure hirni-dicom-converter 
[INFO] ...>run_procedure:452>utils:413,552>run:239,628,460  == Command start (output follows) ===== 
[INFO] hirni-dicom-converter:114>dataset:498>utils:494,482,413,552>containers_run:138>run:588,343  Making sure inputs are available (this may take some time) 
[WARNING] ...>run:588,349>utils:494,482,413,552>get:780,542>gitrepo:328>annexrepo:1352,2358  Running get resulted in stderr output: download failed: Unauthorized
download failed: Unauthorized
download failed: Unauthorized
git-annex: get: 1 failed
 
[ERROR] ...>utils:494,482,413,588  Remote psydata-store not usable by git-annex; setting annex-ignore [get(/tmp/datalad_temp__single_session_dicom2bids5fgc8gsb/code/hirni-toolbox/converters/heudiconv/heudiconv.simg)] 
[INFO] ...>run:628,460  == Command start (output follows) ===== 
ERROR  : Image path code/hirni-toolbox/converters/heudiconv/heudiconv.simg doesn't exist: No such file or directory
ABORT  : Retval = 255
[INFO] ...>run:628,488  == Command exit (modification check follows) ===== 
[INFO] ...>run:699  The command had a non-zero exit code. If this is expected, you can save the changes with 'datalad save -d . -r -F .git/COMMIT_EDITMSG' 
```

- The added log messages in `spec2bids` don't appear at all. How so?
- psydata special remote for hirni-toolbox not accessible. How can that only apply to Github Action while it's not a problem via Travis?

**Comment by bpoldrack** [comment]
Another note: Within the same test run as above, another conversion succeeds, which needs to get the very same container from the very same source. No "download failed: Unauthorized" there.

File: datalad_hirni/tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from datalad import cfg
+
+
+def install_demo_dataset(ds, path, recursive=False):
```
Comment (yarikoptic) [inline datalad_hirni/tests/utils.py:4]: So is this is the fix ? Can't figure it out given amount of description here or in the PR

Comment (bpoldrack) [reply datalad_hirni/tests/utils.py:4]: Part of it. Will close this PR in favor of:
https://github.com/datalad/datalad-extensions/pull/12
and
https://github.com/psychoinformatics-de/datalad-hirni/pull/160

**Pull request closed without merging**